app.js

```diff
@@ -1,5 +1,37 @@
-// import functions and grab DOM elements
+/* eslint-disable quotes */
+let y = Math.floor(Math.random() * 20 + 1); 
```

Comment:
renaming these variables to something that is easy to read, such as randomNumber or generatedNumber would probably make it easier to follow in code later.